go-client/tx_raw_client.go

```diff
@@ -131,6 +143,19 @@ func (c *RawTxClient) BuildTx(ctx context.Context, gasLimit uint64, fees types.C
 		return nil, fmt.Errorf("set empty signature: %w", err)
 	}
 
+	if autoEstimateGas {
```

Comment:
can we move this if together with `if !autoEstimateGas`? so similar code is closer to each other